packages/caip/tsconfig.json

```diff
@@ -2,13 +2,12 @@
   "extends": "../../tsconfig.packages.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "dist"
+    "outDir": "dist",
+    "module": "esnext",
+    "target": "ESNext",
+    "moduleResolution": "Node",
```

Comment:
should just remove `module` from `tsconfig.packages.json` so we use the base `tsconfig.json` module value already specified as `esnext`.

seems like we can also set the base `target` to `esnext` as well.

`moduleResolution` is already specified correctly in base